### DISCUSSION TITLE
Terraform integration with Cloud Connector

### DISCUSSION THREAD

**Discussion by jaaphellemons82** [SAP/terraform-provider-btp#639] [Ideas]
Hi,
although the SAP cloud connector is not directly related to BTP, it is closely coupled. Therefore I am wondering if there are any plans to develop a terraform provider/modules to manage cloud connectors. For example the configuration of subaccounts within the CC. Currently this is a manual task, which we would like to automate.

KR, Jaap

**Comment by lechnerc77** [comment]
Hi @jaaphellemons82 that is indeed an interesting point. The scope of the Terraform provider is defined by the functionality provided by the BTP CLI. 
 
Hence, as of today we do not have any plans on integrating the Cloud Connector with its dedicated REST APIs into the provider. Nevertheless, an interesting point. You can open a [feature request](https://github.com/SAP/terraform-provider-btp/issues/new?assignees=&labels=enhancement%2Cpending-decision&projects=&template=feature_request.yml&title=%5BFEATURE%5D) for that, so we can discuss this topic with the team. However, the current priority 1 is to make the provider GA, so this discussion will probably happen after GA.

**Comment by lechnerc77** [comment]
Feature Request is open: https://github.com/SAP/terraform-provider-btp/issues/734 - check further progress via the issue.
